GitUpKit/Interface/GIGraphView.m

```diff
@@ -74,7 +74,7 @@
 #define CONVERT_X(x) (kSpacingX + (x)*kSpacingX)
 #define CONVERT_Y(y) (kSpacingY + (y)*kSpacingY)
 #define SQUARE(x) ((x) * (x))
-#define SELECTED_NODE_BOUNDS(x, y) NSMakeRect(x - kSpacingX / 2, y - kSelectedLabelMaxHeight / 2, kSelectedLabelMaxWidth + kSpacingX / 2 + 40, kSelectedLabelMaxHeight)
+#define SELECTED_NODE_BOUNDS(x, y) NSMakeRect(x - kSpacingX / 2, y - (kSelectedLabelMaxHeight / 2) - kSubLineWidth, kSelectedLabelMaxWidth + kSpacingX / 2 + 40, kSelectedLabelMaxHeight + (kSubLineWidth * 2))
```

Comment:
I think kSubLineWidth changes the width of the lines between nodes, I don't believe that controls border width. Probably shouldn't use that, but I'm fine with making a new constant that reflects this value.